account_chart_report/wizard/account_report_chart_of_account.py

```diff
@@ -20,25 +20,23 @@
 #
 ##############################################################################
 
-from openerp.osv import fields, orm
+from openerp import fields, models, api
 
 
-class ChartOfAccountsReport(orm.TransientModel):
+class ChartOfAccountsReport(models.TransientModel):
     _name = 'account.print.chart.accounts.report'
     _description = 'Chart of accounts Report'
 
-    domain_char_account = [('parent_id', '=', False)]
-    _columns = {
-        'chart_account_id': fields.many2one('account.account',
-                                            'Chart of Accounts',
-                                            help='Select Charts of Accounts',
-                                            required=True,
-                                            domain=domain_char_account),
-    }
+    chart_account_id = fields.Many2one(
+            'account.account',
+            'Chart of Accounts',
```

Comment:
./account_chart_report/wizard/account_report_chart_of_account.py:31:13: E126 continuation line over-indented for hanging indent

Please remove 4 spaces of indent